cluster/gce/util.sh

```diff
@@ -852,10 +866,7 @@ function create-subnetworks() {
       exit 1
     fi
 
-    if [[ -z ${NODE_IP_RANGE:-} ]]; then
-      echo "${color_red}NODE_IP_RANGE must be specified{color_norm}"
-      exit 1
-    fi
+    assert-node-ip-range
```

Comment:
Correct me if I'm wrong, this section is only called for ENABLE_IP_ALISES=true. Is this assertion necessary if it the variable is set in config-default when aliases are enabled?

Reply:
Yeah seems okay to remove this assertion.

Reply:
Removed.